src/gtfs_flex_validation.py

```diff
@@ -35,58 +42,25 @@ def validate(self) -> tuple[bool, str]:
     # when ready to replace, replace the call in the 
     # above function.
     def is_gtfs_flex_valid(self) -> tuple[Union[bool, Any], Union[str, Any]]:
-        dest = DOWNLOAD_FILE_PATH
         is_valid = False
         validation_message = ''
-        if self.file_relative_path.endswith('.json') or self.file_relative_path.endswith('.zip'):
+        root, ext = os.path.splitext(self.file_relative_path)
+        if ext and ext.lower() == '.zip':
             downloaded_file_path = self.download_single_file(self.file_path)
             try:
+                logger.info(f' Downloaded file path: {downloaded_file_path}')
                 gcv_test_release.test_release(DATA_TYPE, SCHEMA_VERSION, downloaded_file_path)
                 is_valid = True
+                GTFSFlexValidation.clean_up(downloaded_file_path)
```

Comment:
shouldn't this be after try block? in the case where the file is not valid, we will not delete the downloaded file on the current server, which will lead to explosion of local storage.

Reply:
you are correct